### ISSUE TITLE
Cant compile pixi.js.d.ts

### ISSUE THREAD

**Issue by humphrey-parasoft** [pixijs/pixi-typescript#160]
I can't seem to compile the pixi.js.d.ts file from this repository. I'm using grunt-ts 5.5.1 which by default uses the typescript 1.8.9 compiler which gives me 45 syntax errors. I thought that maybe it was an issue with the compiler version so I changed the grunt-ts config to use the latest typescript compiler and got 450 errors. My question is what compiler version should I be using.

**Comment by clark-stevenson** [comment]
I would have thought  2.0.0+ would work.  If I pull the project and run `tsc pixi.js.d.ts` I don't seem to get any errors. I am using typescript `2.3.2` globally.   In your long list of errors, is there any top error, like, It can't find PIXI etc? I am also wondering why it is broken.

**Comment by humphrey-parasoft** [comment]
I found the issue. I ran tsc pixi.js.d.ts and got no errors so looking a little deeper I found somebody on my team had declared PIXI : any in a couple of our other typescript files. But I ran into another issue.  Looking at the pixi.js.d.ts file I found this.

export interface WebGLRendererOptions {

        view?: HTMLCanvasElement;
        transparent?: boolean;
        autoResize?: boolean;
        antialias?: boolean;
        forceFXAA?: boolean;
        resolution?: number;
        clearBeforeRender?: boolean;
        backgroundColor?: number;
        preserveDrawingBuffer?: boolean;
        roundPixels?: boolean;
        legacy?: boolean;
    }

These are supposed to be the parameters passed into the WebGLRenderer constructor but there seems to be a discrepancy between this and the actual API. If you look at http://pixijs.download/dev/docs/PIXI.WebGLRenderer.html you'll notice that the options include width, height and transparent.  I found that the typescript definition actually includes two constructor declarations for WebGLRendererOptions

**Comment by clark-stevenson** [comment]
Ahh thanks for pointing this out, I will take a look. 

The overloaded constructors are I think fine, as the commit below is backwards compatible (so we have 2) but it hurts my head a little. 

https://github.com/pixijs/pixi.js/commit/b8775e180fd7214baf755455f0cda4e2a0d808e5#diff-16d287768c4bce46cc0c29a7c47ea47aL55

However, you are totally correct! `width` and `height` should be found on the `WebGLRendererOptions`.  If you want to make a PR I will happily merge. I have a few weeks of changes to push as well (blocked by #156) so I can't push immediately but have made the change. 

👍

**Comment by humphrey-parasoft** [comment]
I'm gonna close the issue since everything is working on my end now. From your last comment I wasn't sure if you wan't me to make a pull request or If you've already made the change and will update at a later time

**Comment by clark-stevenson** [comment]
👍
